projects/plugins/boost/package.json

```diff
@@ -0,0 +1,116 @@
+{
+	"name": "jetpack-boost",
+	"version": "1.1.0",
+	"description": "Boost your WordPress site's performance, from the creators of Jetpack",
+	"directories": {
+		"test": "tests"
+	},
+	"dependencies": {
+		"@wordpress/components": "^14.1.2",
+		"@wordpress/element": "^3.1.1",
+		"jetpack-boost-critical-css-gen": "git+ssh://git@github.com/automattic/jetpack-boost-critical-css-gen.git#trunk"
```

Comment:
Should this be imported into the monorepo too? It'd probably go in `projects/js-packages/jetpack-boost-critical-css-gen/`.

Reply:
Probably should, but we can do it separate from this PR.